src/syntax/errors/list.rs

```diff
@@ -69,17 +72,17 @@ pub fn get_error_desc(err: &ErrorKind) -> (&'static str, String, &'static str) {
         }
         ErrorKind::ForbiddenWhitespace => (
             "E0007",
-            "keyword cannot end with a whitespace".to_owned(),
+            "Keyword cannot end with a whitespace".to_owned(),
             "",
         ),
-        ErrorKind::ForbiddenCallee => (
+        ErrorKind::ForbiddenCaller => (
             "E0008",
-            "a callee has to be a simple identifier".to_owned(),
+            "The caller has to be a simple, upper-case, identifier".to_owned(),
             "",
         ),
         ErrorKind::ForbiddenKey => (
             "E0009",
-            "a key has to be a simple identifier".to_owned(),
+            "A key has to be a simple identifier".to_owned(),
```

Comment:
Since you're changing it already: `The key...` or `Keys...` both sound better to me.